apps/aecore/src/aec_base58c.erl

```diff
@@ -107,6 +123,11 @@ pfx2type(<<"nm">>) -> name;
 pfx2type(<<"st">>) -> state;
 pfx2type(<<"pi">>) -> poi.
 
+type2size(poi) -> not_set;
+type2size(name) -> not_set;
+type2size(transaction) -> not_set;
+type2size(_) -> 32.
```

Comment:
May I suggest explicitly listing each and every one of the types? This catch all may lead to surprising errors - especially if new types are introduced or size of any of the existing types changes.